components/brave_wallet/browser/network_manager.cc

```diff
@@ -775,7 +776,7 @@ mojom::NetworkInfoPtr NetworkManager::GetKnownChain(const std::string& chain_id,
 
 mojom::NetworkInfoPtr NetworkManager::GetCustomChain(
 
-    const std::string& chain_id,
+    std::string_view chain_id,
     mojom::CoinType coin) {
   const base::Value::List* custom_list = GetCustomNetworksList(prefs_, coin);
```

Comment:
Could you please aslo remote this empty line

Reply:
Fixed